cli/src/cli-rpc-ops.c

```diff
@@ -5662,9 +5683,12 @@ cmd_profile_volume_brick_out(dict_t *dict, int count, int interval)
     }
 
     cli_out(" ");
-    cli_out("%12s: %" PRId64 " seconds", "Duration", sec);
+    cli_out("%12s: %" PRId64 " seconds", "Start time", start_time);
+    cli_out("%12s: %" PRId64 " seconds", "End time", end_time);
     cli_out("%12s: %" PRId64 " bytes", "Data Read", r_count);
     cli_out("%12s: %" PRId64 " bytes", "Data Written", w_count);
+    cli_out("%12s: %" PRId64, "FOPS/S", fop_hits_per_sec);
```

Comment:
There's a missing argument for the format string. I would also put the "FOPS/S" in lowercase.

Reply:
done